roles/container_runtime/templates/overlay2_dss.j2

```diff
@@ -0,0 +1,12 @@
+# Edit this file to override any configuration options specified in
+# /usr/lib/docker-storage-setup/docker-storage-setup.
+#
+# For more details refer to "man docker-storage-setup"
+DEVS={{ container_runtime_docker_storage_setup_device }}
+VG=docker_vg
+DATA_SIZE=99%VG
+STORAGE_DRIVER="overlay2"
+CONTAINER_ROOT_LV_NAME="docker-root-lv"
+CONTAINER_ROOT_LV_SIZE="100%FREE"
+CONTAINER_ROOT_LV_MOUNT_PATH="{{ docker_storage_path }}"
+EXTRA_STORAGE_OPTIONS="--storage-opt overlay2.override_kernel_check=true --storage-opt overlay2.size={{ docker_storage_size }} --graph={{ docker_storage_path}} "
```

Comment:
We should try to make these all variables instead of hard-coding.
extra_storage_options should take a list of options.

```
eso:
  - '--storage-opt overlay2.override_kernel_check=true'
  - "--graph={{ docker_storage_path}}"
```

This will be easier for the user to override, and we don't have to implement any fancy logic other than joining the list items.

Reply:
Will do.